web/src/portico/team.ts

```diff
@@ -1,9 +1,31 @@
 import $ from "jquery";
 import _ from "lodash";
 
-import {page_params} from "../page_params";
+import * as blueslip from "../blueslip";
+
+const all_repository_names = [
+    "zulip",
+    "zulip-desktop",
+    "zulip-mobile",
+    "python-zulip-api",
+    "zulip-js",
+    "zulipbot",
+    "zulip-terminal",
+    "zulip-ios-legacy",
+    "zulip-android",
+] as const;
+type RepositoryNameType = (typeof all_repository_names)[number];
+
+export type Contributor = {
+    avatar: string;
+    email: string;
+    github_username?: string;
+    name: string;
+} & {
+    [K in RepositoryNameType]: number;
```

Comment:
Not every repository has an entry: `[K in RepositoryName]?: number`.

Reply:
Fixing this reveals that the existing logic has a lot of comparisons of a thing that may be undefined, fun.